NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Ticket 10- Create Category

Description
Added a feature to create new categories and some styling fixes. [Trello Ticket #10](https://trello.com/c/k6DujJT8)

Type of Change
-Added new, non-breaking feature.
-Styling fixes.

Testing Instructions


1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout sf-ticket-10
5. If you don't see the most recent version of my files, git pull origin sf-ticket-10
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-full-stack-force/blob/main/SQL/02_Tabloid_Seed_Data.sql)
9. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
10. In your terminal, navigate to where your package.json file is in you client folder. Run "npm start" in your terminal.
11. If not logged in, you may login with any user in the database. I have been using [foo@bar.com](mailto:foo@bar.com) for the email parameter. Leave the password text box blank. Click "Login".
12. Click on “Category Management” in the nav bar, this will redirect you to the /categories page.
13. You should be redirected to a page where all categories are listed out in alphabetical order on the left hand side of your screen.
14. Click on the button "Create New Category" in the top right to be redirected to a form to create a new category.
15. For the new category name, type in "Brand New Category," and click submit.
16. You will be redirected back to the categories page, and you should see the newly created category at the top of the page.

- Brand New Category
- Cooking
- Cthulhu Sightings
- Entertainment
- History
- Home and Garden
- Music
- Politics
- Science
- Technology

17. Repeat steps 14 and 15, but this time fill in "Bad Bad Category", and click cancel. You should be redirected back to the categories page and nothing should have been created

Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
